core-services/prow/02_config/redhat-appstudio/build-service/_pluginconfig.yaml

```diff
@@ -59,6 +59,8 @@ plugins:
     - owners-label
     - wip
     - yuks
+  redhat-appstudio/build-definitions:
```

Comment:
I guess adding it to a separate file (`core-services/prow/02_config/redhat-appstudio/build-definitions/_pluginconfig.yaml`) could eliminate some of the errors

```yaml
plugins:
  redhat-appstudio/build-definitions:
    plugins:
    - jira
```